### DISCUSSION TITLE
Weird 200W consumption floor on sunny days

### DISCUSSION THREAD

**Discussion by markusdd** [hoylabs/OpenDTU-OnBattery#984] [Q&A] [answered]
Currently for the last approx 3 days I see an interesting effect that I cannot really explain:

![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/25175069/3cb1cb8a-2bbd-4df8-a575-6f6d24ab40c4)

The power limiter tries to cancel out any grid consumption (base load is usually around 500-600W), but since it was so sunny, despite the inverter limit being set, I still get around 200-300W consumption, the rest seems to go straight into the battery.

The only way I can explain this is that the path to the inverter has higher resistance than charging the battery. Which would be kind of surprising to be honest, I kept the lines pretty short and everything (- and +) collects onto a single post each. I have also not seen this before, cancelling out to about 0W worked almost perfectly.

I will observe what happens when the battery goes to almost full, as the current then should drop, but does anyone have aan idea what is going wrong here? Any setting I might be able to tweak?

**Comment by markusdd** [comment]
here is a picture of about 2 weeks ago from a sunny day. Inverter power tracks inverter limit perfectly and when there is enough solar power the consumption goes flat to almost perfect 0.
![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/25175069/60585a00-67d8-4134-bfa4-4d3426130b35)

**Reply by Manos1966** [reply]
@markusdd ist das Grafana? Hast du ein paar Links mit Erlaeuterungen wie man eine solche Grafik herstellen kann? 
Ich bin sicher, viele Anwender wuerden gerne so eine Tabelle haben...

**Reply by markusdd** [reply]
Mein Grafana-Dashboard ist open source.
Das ist zwar noch nicht die 'onBattery' variante die ich da eingestellt hab, aber das grundprinzip ist das gleiche:
https://github.com/markusdd/solar_dashboard

EDIT: Habe auch jetzt die aktuelle Variante hochgeladen.

**Reply by Manos1966** [reply]
@markusdd Much appreciated! 
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Grafana-Dashboard

**Comment by markusdd** [comment, marked as answer]
Ok, found the issue lol.

The 2 string inputs to the inverter were completely assymettric. ironically, I checked all connections and they were tight, soo noothing loose.

but I wiggeled everything a bit and now it's point on again. Too me this means there must be some slight contact corrosion gooing on, maybe accelerated by the higher currents in the last days.
Will observe and maybe use some electric grease if the issue comes back.

**Comment by schlimmchen** [comment]
Took me a while to understand what your issue was, but yeah, I can see it now. The inverter is set to a certain limit, but it does not produce as much.

I had the exact same learning curve. My setup was working fine, then some day I saw the inverter was not producing as much as the limit allowed. The power produced by channel was wildly different and fluctuating a lot. I also thought that I had tightened all screws properly. And I had, but on some only one out of two screws reached the cable. See this image:

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/3578416/c6aedbf6-b17f-4736-9b82-50bc1cc0f4f7)

The screw below is down a lot more than the screw above, and that's because the cable does not reach the second screw. I removed the blue collar from the ferrule, such that the cable went down further and the second screw could reach it. There were other connections and screws that loosened a little over time. After going over all of them, the issue was gone.

I thought this was the perfect opportunity to share this experience with a little more detail. I hope I can find and link to this discussion the next time I need it :wink: 

Regarding contact grease: I also thought about it, and learned that it is actually not conductive. One uses contact grease to lubricate fasteners and prevent corrosion, but you actually don't put it between contact surfaces that shall conduct electricity. At least that's what I researched a couple of months ago. I don't use any and rewired the whole setup into a new cabinet recently. After learning that connections must be very good, I took care to double check every single one and I had no issue firing up the new wiring.

**Comment by Manos1966** [comment]
> The 2 string inputs to the inverter were completely assymettric

And in my case, four (I was too busy one Sunday afternoon to put things together again that I mixed a set of cables): 
![901 A difference](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/d7cba23d-0510-47fa-b3a0-089eb69daecb)
